scripts/components/icon-toggle/docs/story.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { IconToggle } from '../icon-toggle';
+import { icons } from '../../../editor/icons/icons';
+import readme from './readme.mdx';
+
+export default {
+	title: 'Options/IconToggle',
+	parameters: {
+		docs: {
+			page: readme
+		}
+	},
+};
+
+export const component = () => {
+	const [current, setCurrent] = useState(false);
+
+	return (
+		<div style={{width: '20rem'}}>
+			<IconToggle
+				icon={icons.width}
+				label='Full width'
+				checked={current}
+				onChange={(value) => setCurrent(value)}
+			/>
+
+			<br />
+
+			<IconToggle
+				icon={icons.width}
+				label='Full width'
+				checked={current}
+				onChange={(value) => setCurrent(value)}
+				help='Help text here.'
```

Comment:
Not translatable?

Reply:
This question applies to line 22, 31, and 34 😄

Reply:
It's just a storybook example, while it can be translated, didn't see the need to do it here as I found thee translations to be a bit wonky in storybook. If you think they should be there, I'll add them 😄

Reply:
I've checked out other storybook examples, none of them use localization, just straight-up strings. 🤷

Reply:
storybook doesn't need translations it is dummy data

Reply:
Added them 😄